dxr/mime.py

```diff
@@ -4,17 +4,18 @@
 def icon(path):
     """Return the basename (no extension) of the icon file to use for a path."""
     root, ext = splitext(path)
-    return ext_map.get(ext[1:], 'unknown')
+    return ext_map.get(ext[1:].lower(), 'unknown')
 
 
 def is_text(data):
     # Simple stupid test that apparently works rather well :)
     return '\0' not in data
 
+
 def is_image(path):
     """Determine whether the path is an image."""
```

Comment:
We should probably make the meaning of this more specific, since SVG has shown a use case for said specificity.